EqualityInformationApi.Tests/DynamoDbMockWebApplicationFactory.cs

```diff
@@ -1,15 +1,15 @@
 using Amazon.DynamoDBv2;
 using Amazon.DynamoDBv2.DataModel;
 using Amazon.DynamoDBv2.Model;
-using BaseApi.V1.Infrastructure;
+using EqualityInformationApi.V1.Infrastructure;
 using Hackney.Core.DynamoDb;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Mvc.Testing;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using System.Collections.Generic;
 
-namespace BaseApi.Tests
+namespace EqualityInformationApi.Tests
 {
     public class DynamoDbMockWebApplicationFactory<TStartup>
```

Comment:
It looks like we have 2 of the same files (this one and the `AwsMockWebApplicationFactory`)